src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -330,6 +330,33 @@ public static boolean isOlderThanAYear(Date compareDate) {
     }
     
     /**
+     * Returns true if the {@code time} falls within the last hour.
+     * That is exactly one hour or less from the current time but earlier than current time.
+     * Precision is at millisecond level.
+     */
+    public static boolean isWithinPastHour(Date time) {
+        return isWithinHour(time, new Date());
+    }
+    
+    /**
+     * Returns true if the {@code time1} falls within past 1 hour of {@code time2}.
+     * That is exactly one hour or less from time2 but earlier than time2.
+     * Precision is at millisecond level.
+     */
+    public static boolean isWithinHour(Date time1, Date time2) {
```

Comment:
Hmmm now the method name itself is not so clear (the method header helps, but don't count so much on it). `isWithinHour` can mean `time1` can be before or after `time2` returns true, while the intention here is strictly `time1` before `time2`.